backend/src/main/java/com/google/sps/datastore/ParticipantDatastore.java

```diff
@@ -90,16 +93,23 @@ private static Participant getParticipantFromEntity(@Nonnull Entity entity) {
         (long) entity.getProperty(PROPERTY_ENDTIMEAVAILABLE),
         ((Long) entity.getProperty(PROPERTY_DURATION)).intValue(),
         (long) entity.getProperty(PROPERTY_MATCHID),
+        MatchStatus.forIntValue(((Long) entity.getProperty(PROPERTY_MATCHID)).intValue()),
```

Comment:
(long) instead of (Long)?

Reply:
Since it's stored as an int, and I can't directly cast to an int, I have to first convert it to a Long and then get the int value. Or I could store it as a long instead of an int so I could just do (long), but I think that'd be more complicated because conversion between ints and enums seems more natural. What do you think?

Reply:
Ok, I see, it's stored as a Long and so a direct case to int is impossible. Ok to leave as is